src/Chapter1.hs

```diff
@@ -509,7 +509,7 @@ branches because it is an expression and it must always return some value.
 -}
 closestToZero :: Int -> Int -> Int
 closestToZero x y = error "closestToZero: not implemented!"
```

Comment:
As I mentioned before, you need to replace the part with `error "..."` with your implementation for tests to pass